physionet-django/project/modelcomponents/publishedproject.py

```diff
@@ -201,6 +201,9 @@ def has_access(self, user):
         if self.deprecated_files:
             return False
 
+        if not self.allow_file_downloads:
+            return False
```

Comment:
To me this if statement feels like it is misinterpreting the original intent of the `has_access` method. I think it could be confusing in future. 

The method is asking the question "Does the user have access to the files". If `allow_file_downloads` is False, the answer might still be "yes, they do have access to the files but only in the cloud environment".

Perhaps we could have an argument that specifies the type of access? e.g. `access_type="download"`? Or a separate "can_download" method?

Reply:
@bemoody has convinced me that this isn't an issue for now, so I think we can ignore my comment.